cockroachcloud/export-metrics.md

```diff
@@ -125,7 +125,7 @@ Perform the following steps to enable metrics export from your {{ site.data.prod
     {% include_cached copy-clipboard.html %}
     ~~~shell
     curl --request POST \
-      --url https://cockroachlabs.cloud/api/v1/clusters/{cluster_id}/metricexport \
+      --url https://cockroachlabs.cloud/api/v1/clusters/{cluster_id}/metricexport/cloudwatch \
       --header "Authorization: Bearer {secret_key}" \
       --data '{"cloudwatch":{"target_region": "{aws_region}", "role_arn": "arn:aws:iam::{role_arn}:role/CockroachCloudMetricsExportRole", "log_group_name": "{log_group_name}"}}'
```

Comment:
The "cloudwatch" config is unnested now, so this line can be replaced with `--data '{"target_region": "{aws_region}", "role_arn": "arn:aws:iam::{role_arn}:role/CockroachCloudMetricsExportRole", "log_group_name": "{log_group_name}"}'`